server/config.ts

```diff
@@ -125,6 +126,12 @@ export default {
           completedCases: Number(get('PP_MY_CASES_PAGE_SIZE', '500')),
           cancelledCases: Number(get('PP_CANCELLED_CASES_PAGE_SIZE', '500')),
         },
+        sp: {
+          myCases: get('SP_MY_CASES_PAGE_SIZE', '500'),
```

Comment:
there is an unintended change here that the vars are no longer converted to `Number`